bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClient.java

```diff
@@ -687,7 +688,7 @@ void connectIfNeededAndDoOp(GenericCallback<PerChannelBookieClient> op) {
     void writeLac(final long ledgerId, final byte[] masterKey, final long lac, ByteBufList toSend, WriteLacCallback cb,
             Object ctx) {
         final long txnId = getTxnId();
-        final CompletionKey completionKey = new V3CompletionKey(txnId,
+        final CompletionKey completionKey = new TxnCompletionKey(txnId,
```

Comment:
why did you change this name ?V3CompletionKey > TxnCompletionKey

Reply:
The `V3CompletionKey` may mislead. In this PR, it's used for the v2 batch read. The V3CompletionKey uses txnId to trace, so I change the name to TxnCompletionKe.